<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TF2 Loadout Maker 5000</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../game.ico</normaloff>../game.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="merc_cbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Scout</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Scout</string>
    </property>
    <item>
     <property name="text">
      <string>Scout</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Soldier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pyro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Demoman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heavy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Engineer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sniper</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spy</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="game_cbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Vanilla</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Merc</string>
    </property>
    <item>
     <property name="text">
      <string>Vanilla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Classic</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Merc:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Game:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>240</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Game:
Merc:
Primary:
Secondary:
Melee:
PDA:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>135</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disallow cosmetic reskins</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>59</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(disabled for the sake of my sanity)</string>
    </property>
   </widget>
  </widget>
  <action name="actionDisallow_cosmetic_reskins">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disallow cosmetic reskins</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
